source/administration/config-settings.rst

```diff
@@ -4037,6 +4035,104 @@ Changes to this setting require a server restart before taking effect.
 Experimental Settings only in config.json
 -----------------------------------------
 
+Audit settings
+~~~~~~~~~~~~~~~
+The audit settings output audit records to syslog (local or remote server via TLS) and/or to a local file. Both are disabled by default.
+
+Syslog configuration options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
Wondering if this needs a description?

Reply:
I think both syslog and file config options would benefit from a description; I'll ping you when I've added them. :)

Reply:
@wiersgallak, I've added descriptions.